mne/decoding/transformer.py

```diff
@@ -12,42 +12,75 @@
 from .. import pick_types
 from ..filter import filter_data, _triage_filter_params
 from ..time_frequency.psd import _psd_multitaper
-from ..externals import six
-from ..utils import _check_type_picks
+from ..utils import _check_type_picks, requires_sklearn, check_version
+from ..io.pick import pick_info, _pick_data_channels, _picks_by_type
+from ..cov import _undo_scaling_array, _apply_scaling_array
 
 
-class Scaler(TransformerMixin):
-    """Standardize data across channels.
+@requires_sklearn
```

Comment:
Only decorate test functions with `requires_*` functions, not classes. This is probably where your error comes from